make/lib/Lib-java.base.gmk

```diff
@@ -176,6 +182,7 @@ ifeq ($(OPENJDK_TARGET_OS_TYPE), unix)
 
   endif
 endif
+endif
```

Comment:
A comment to help match this with the opening `ifeq` would be helpful.

Reply:
added